### PR TITLE
Adds commit to startup message

### PR DESCRIPTION
**Pull request by darrylmasson**
Rather than only logging the commit on startup, now also log at the top of every new logfile for easier finding.

### CODE REVIEW

**Comment by darrylmasson** [comment]
Does require https://github.com/XENONnT/daqnt/pull/85

File: dispatcher/dispatcher.py [hunk 1]
```diff
@@ -27,8 +36,10 @@ def setup():
     daq_config = json.loads(config['MasterDAQConfig'])
     control_mc = daqnt.get_client('daq')
     runs_mc = daqnt.get_client('runs')
-    logger = daqnt.get_daq_logger(config['LogName'], level=args.log, mc=control_mc)
+    logger = daqnt.get_daq_logger(config['LogName'], level=args.log, mc=control_mc, log_on_rotate=f'Commit {commit}')
```
Comment (JoranAngevaare) [inline dispatcher/dispatcher.py:39]: unless we change the PR, it would be `opening_message`

**Review by JoranAngevaare — COMMENTED** [review]
looks good

File: dispatcher/dispatcher.py (hunk 1 above)
Comment (darrylmasson) [reply dispatcher/dispatcher.py]: Have to pad my commit count somehow

**Review by JoranAngevaare — APPROVED** [review]

**Pull request merged**